libraries/AP_Tuning/AP_Tuning.h

```diff
@@ -90,13 +94,15 @@ class AP_Tuning
     void save_parameters(void);
     void revert_parameters(void);
     const char *get_tuning_name(uint8_t parm);
+    void check_controller_error(void);
 
 protected:
     // virtual functions that must be implemented in vehicle subclass
     virtual AP_Float *get_param_pointer(uint8_t parm) = 0;
     virtual void save_value(uint8_t parm) = 0;
     virtual void reload_value(uint8_t parm) = 0;
     virtual void set_value(uint8_t parm, float value) = 0;
+    virtual float controller_error(uint8_t parm) = 0;
```

Comment:
Maybe I'm missing something, but since this isn't implemented here, doesn't Copter also need to have it implemented? Otherwise it will crash right? I though it wouldn't even link.